src/arrays.js

```diff
@@ -1,61 +1,93 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index < array.length) {
+    return array[index];
+  }
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
```

Comment:
There is a native operator in JS which can do what you need here: remainder (%). Its like divide except it will return the remainder of the division instead of a decimal (EG 7 % 5 = 2)

```suggestion
  return array[index % array.length];
```